### PR TITLE
appveyor generates python wheels with pcl dlls included for windows

### PR DESCRIPTION
**Pull request by martinResearch**
keeping a copy of  the generated wheel in the appveyor's artefacts. This will ease the installation in windows, using the wheel instead of installing the windows toolkits etc. the wheel for windows for python 3.6 is available here https://ci.appveyor.com/project/martinResearch/python-pcl/build/job/94euli1pm58nfdox/artifacts

### CODE REVIEW

**Comment by martinResearch** [comment]
we could then upload the windows wheels to PyPi to make the installation even easier

**Comment by martinResearch** [comment]
@nzjrs does it look good for a merge ?

**Comment by martinResearch** [comment]
one note on that: due to the limitation of AppVeyor’s build system, the wheel has openMP disabled.

File: setup.py [hunk 1]
```diff
@@ -622,8 +624,27 @@ def pkgconfig_win(flag, cut):
     else:
         print('no pcl install or pkg-config missed.')
         sys.exit(1)
+		
+		
+
+
+		
+if platform.system() == "Windows":
```
Comment (Sirokujira) [inline setup.py]: Please write it in the if - else you defined earlier.
Please Add Windows logic to the end of if.(line 505)
Please Add MacOSX/Linux logic to the end of else.(line 627)

File: readme.rst [hunk 2]
```diff
@@ -225,7 +245,7 @@ before Install module
 `Python Version use VisualStudio Compiler <https://wiki.python.org/moin/WindowsCompilers>`_ 
 
 set before Environment variable
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~~~~~
```
Comment (Sirokujira) [inline readme.rst]: Please correct the number of ^ to match the above characters.

File: readme.rst [hunk 3]
```diff
@@ -187,9 +187,29 @@ circumvent:
...
+			pip install python_pcl-XXX.whl
+	
+		
+Compiling the binding from source
+^^^^^^^^^^^^^^^^^^^^^		
+
+	If the method using the procompiled wheel does not work you can compile the binding from the source.
+	
+before Install module
+~~~~~~~~~~~~~~~~~~~
```
Comment (Sirokujira) [inline readme.rst]: Please correct the number of ^ to match the above characters.

File: readme.rst [hunk 4]
```diff
@@ -187,9 +187,29 @@ circumvent:
...
+	3. Go in the artfacts section for that job and download the wheel (the file with extension whl)
+	4. In the command line, move to your download folder and run the following command (replacing XXX by the right string)	
+	
+.. code-block:: none
+
+			pip install python_pcl-XXX.whl
+	
+		
+Compiling the binding from source
+^^^^^^^^^^^^^^^^^^^^^
```
Comment (Sirokujira) [inline readme.rst]: Please correct the number of ^ to match the above characters.

File: readme.rst [hunk 5]
```diff
@@ -187,9 +187,29 @@ circumvent:
 Windows
 -------
 
-before Install module
+Using pip with a precompiled wheel
```
Comment (Sirokujira) [inline readme.rst]: Please correct the number of ^ to match the above characters.

**Comment by Sirokujira** [comment]
Thanks for the PR.

**Pull request merged**